src/components/Comment/index.tsx

```diff
@@ -13,10 +13,42 @@ import {
 } from '../../actions/comments';
 import { LayoutController } from '../../utils/layout';
 import { getNextReplyId } from '../../utils/sequences';
-import CommentReplyComponent from '../CommentReply';
+import CommentReplyComponent, { addRemoteWarningMessage as addRemoteReplyWarningMessage } from '../CommentReply';
```

Comment:
I think you should consider renaming that other function directly where it’s defined, or better yet move those two action creators within the Redux code. It feels like an anti-pattern for one component to import Redux code from another component.